_posts/en/newsletters/2019-10-30-newsletter.md

```diff
@@ -0,0 +1,152 @@
+---
+title: 'Bitcoin Optech Newsletter #70'
+permalink: /en/newsletters/2019/10/30/
+name: 2019-10-30-newsletter
+slug: 2019-10-30-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the latest C-Lightning release,
+requests help testing a Bitcoin Core release candidate,
+describes discussions about simplified LN commitments using CPFP
+carve-out, and summarizes several top-voted questions and answers from
+the Bitcoin StackExchange.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Upgrade to C-Lightning 0.7.3:** this latest [release][c-lightning
+  0.7.3] adds support for a PostgresSQL backend, makes it possible to
```

Comment:
s/PostgresSQL/PostgreSQL/

Reply:
possibly worth mentioning "in addition to SQLite3" (IIRC)

Reply:
Thanks for catching that.  A note for anyone else reviewing: this is wrong in the C-Lightning release notes; I checked that the spelling @jonatack proposes is the same as being used by the [upstream PostgreSQL project](https://www.postgresql.org/).